[TC-9408] Rename `lastUpdatedSince` filter to `lastUpdatedAfter` (#84)
tradecloud/tradecloud-docs-api-v2: api/webhook-vs-polling.md

#### Step 1. Fetch updated orders or shipments periodically

Fetch every polling period, typically 5 minutes, all orders or shipments which are new or changed after last date time.

> * Use the latest `lastUpdatedAt` from previous poll request in the `lastUpdatedAfter` filter, which will fetch only updated orders after last date time.

Reviewer: ```suggestion
* Use the latest `lastUpdatedAt` from previous poll request in the `lastUpdatedAfter` filter, which will only return orders that are updated after this date time.
```
